src/hooks/useQueryParams.ts

```diff
@@ -0,0 +1,31 @@
+import { useCallback } from 'react';
+
+const useQueryParams = () => {
+  const getQueryParam = (queryName) => {
+    const queryParams = new URLSearchParams(window.location.search);
+    const queryParam = queryParams.get(queryName);
+    if (!queryParam) return [];
+
+    return queryParam?.split(',');
+  };
+
+  const removeSharedResultsQueryParams = useCallback(() => {
+    const updatedUrl = new URL(window.location.href);
```

Comment:
**suggestion**: I wonder if we should use `quizBasePath` prop like in my PR https://github.com/Constructor-io/constructorio-ui-quizzes/pull/145/files#diff-4ca8c4ece22292244e023432ad4ad692edfa1d8e0a64e44537935be42225251bR115 if supplied

Reply:
but using window location href could be the fallback behavior though